src/lib/sidenav/sidenav.spec.ts

```diff
@@ -251,6 +252,55 @@ describe('MdSidenav', () => {
     });
   });
 
+  describe('focus trapping behavior', () => {
+    let fixture: ComponentFixture<SidenavWitFocusableElements>;
+    let testComponent: SidenavWitFocusableElements;
+    let sidenav: MdSidenav;
+    let link1Element: HTMLElement;
+    let link2Element: HTMLElement;
+
+    beforeEach(() => {
+      fixture = TestBed.createComponent(SidenavWitFocusableElements);
+      testComponent = fixture.debugElement.componentInstance;
+      sidenav = fixture.debugElement.query(By.directive(MdSidenav)).componentInstance;
+      link1Element = fixture.debugElement.query(By.css('.link1')).nativeElement;
+      link2Element = fixture.debugElement.query(By.css('.link1')).nativeElement;
+      link2Element.focus();
```

Comment:
Can we call these something like `firstFocusableElement` and `lastFocusableElement`?